Cargo.toml

```diff
@@ -14,7 +14,7 @@ exclude = ["tests/data/*"]
 [dependencies]
 byteorder = "1.2.7"
 dbase = "0.0.4"
-geo-types = {version = "0.4.3", optional = true}
+geo-types = {version = ">=0.4.0, <=0.6.0", optional = true}
```

Comment:
I think it should be ">=0.4.0, <0.7.0" so that bug fixes releases of geo-types 0.6.x qualify as compatible